packages/core/test/integration/query-interface.test.js

```diff
@@ -21,6 +21,25 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     await Support.dropTestSchemas(this.sequelize);
   });
 
+  if (dialect.supports.schemas) {
```

Comment:
Can you move this test to the other file so they are all in the same place?

Reply:
I'd rather not for right now as that test file is strictly for the TS QI methods - I'd like to keep the current JS QI untouched and move tests and source code over to its TS equivalents in a piecemeal fashion

Reply:
Sure, sounds reasonable